### PR TITLE
refac(*): improving the introduction and getting started pages

### PR DESCRIPTION
**Pull request by lomudi**
## Description

- Improving the introduction and getting started pages in order to make it more clear to a new user.
- Removing unnecessary link to the `public plugin guidelines` in the menu.

## Known issues

nope.

## Checklist

* [x] PR is scoped to one task
* [ ] Tests are included
* [x] Documentation is included

* [x] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

File: book/getting-started/zapp-plugins.md [hunk 1]
```diff
@@ -4,20 +4,35 @@ New plugins for the Zapp platform can be created as native iOS (swift, objective
...
-## How are plugins deployed and submitted to Zapp?
+Open source tools, examples, and live plugins can be found in [GitHub](https://github.com/topics/zapp-plugin) for your use.
+
+## Plugin development flow
+
+**1. Create your project**
+
+The first step in creating a new plugin is to create the plugin project. Our recommendation is first to create the project with the most basic relevant implementation (`hello world`), then make sure it successfully builds inside Zapp before moving to the full implementation. 
+
+In the creation of your project you can use one of our [examples](https://github.com/applicaster/zapp-plugins-examples), [live open source plugins](https://github.com/topics/zapp-plugin), or [Xcode templates](https://github.com/applicaster/zapp-plugins-ios-templates).
```
Comment (liviur) [inline book/getting-started/zapp-plugins.md]: add it as a bullet list please

File: book/getting-started/zapp-plugins.md [hunk 2]
```diff
@@ -4,20 +4,35 @@ New plugins for the Zapp platform can be created as native iOS (swift, objective
...
+
+**2. Publish to Zapp** 
+
+Each plugin should be pushed to its own git repository and deployed (preferably by CI) to a dependancy manager; CocoaPods for [iOS](/getting-started/ios-podspec.md), Maven for [Android](/getting-started/set-up-bintray-and-circle-ci.md), and NPM for React-Native and Datasource providers.
+
+Then publish the [Zapp plugin manifest](/zappifest/plugins-manifest-format.md) (zappifest), which is a JSON that declares the plugin getaways (Class name, Pod/Maven/NPM, and more) and enables the plugin developer to add custom plugin configuration fields in Zapp. You can do it easliy by using the [zappifest cli](https://github.com/applicaster/zappifest).
+
+**3. Build your app and download the development project**
+
+In order to enable the plugin developer to develop the plugin in the most realistic environment, we have created the *Zapp development project*. Follow [this](/getting-started/download-development-project.md) guide to build your app and generate the development project.
```
[the remark this answers is not in the document]
Comment (lomudi) [reply book/getting-started/zapp-plugins.md]: 👍

File: book/getting-started/zapp-plugins.md (hunk 1 above)
Comment (lomudi) [reply book/getting-started/zapp-plugins.md]: 👍

**Comment by lomudi** [comment]
@liviur comments where addressed.

**Pull request merged**